nf_core/modules/lint/main_nf.py

```diff
@@ -120,12 +120,6 @@ def check_script_section(self, lines):
     """
     script = "".join(lines)
 
-    # check for software
-    if re.search("\s*def\s*software\s*=\s*getSoftwareName", script):
```

Comment:
Can we still have a regex looking for the new code and warn if it hasn't been provided?

Reply:
not sure if this is still needed as the modules CI will now complain about missing or ill-formatted `versions.yml`

Reply:
Sorry, I meant linting somehow for the new code to check if a software version is being recorded somehow?